UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -162,7 +162,19 @@ public enum RoomEntryFlags : uint
     /// </summary>
     public UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>> Sequences { get; private set; } = new UndertaleSimpleList<UndertaleResourceById<UndertaleSequence, UndertaleChunkSEQN>>();
 
+    /// <summary>
+    /// Call <see cref="OnPropertyChanged(string)"/> for <see cref="BGColorLayer"/> to update the room background color.<br/>
+    /// Used for GMS 2 rooms.
+    /// </summary>
     public void UpdateBGColorLayer() => OnPropertyChanged("BGColorLayer");
+
+    /// <summary>
+    /// Order the room layers list by depth.
+    /// </summary>
+    /// <param name="selectedLayer">
+    /// Optional argument.<br/>
+    /// Should be present if called from the room editor to re-select the selected layer.
+    /// </param>
```

Comment:
```suggestion
    /// <summary>
    /// Orders the room layers list by depth.
    /// </summary>
    /// <param name="selectedLayer"> Should be present if called from the room editor to re-select the selected layer. </param>
```
Also personally don't quite the selectedLayer's description.